core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -358,11 +363,11 @@ object AdminUtils extends Logging {
     zkUtils.zkClient.exists(getTopicPath(topic))
 
   def getBrokersAndRackInfo(zkUtils: ZkUtils, rackAwareMode: RackAwareMode = RackAwareMode.Enforced,
-                            brokerList: Seq[Int] = Seq()): (Seq[Int], Map[Int, String]) = {
+                            brokerList: Option[Seq[Int]] = None): (Seq[Int], Map[Int, String]) = {
```

Comment:
What is the advantage of using `Option` here? It will require caller to do `Some(list)`.